From 980d48f93204a3bb8e0d0c0bb4c58635aa2cc425 Mon Sep 17 00:00:00 2001
From: Jesse Vincent <jesse@bestpractical.com>
Date: Sat, 7 Mar 2015 12:48:17 -0800
Subject: [PATCH] Drop the "internal formatter" message from info to debug.

An informational message added in 40ba502 appears rather frequently
in normal RT operation (once per commandline script in some cases)
when RT is run with the default logging level 'info'.
---
 lib/RT/Interface/Email.pm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/RT/Interface/Email.pm b/lib/RT/Interface/Email.pm
index 4fdae1b..ef618e6 100644
--- a/lib/RT/Interface/Email.pm
+++ b/lib/RT/Interface/Email.pm
@@ -1835,7 +1835,7 @@ sub _HTMLFormatter {
     # Always fall back to core, even if it is not listed
     for my $prog (@order) {
         if ($prog eq "core") {
-            RT->Logger->info("Using internal Perl HTML -> text conversion");
+            RT->Logger->debug("Using internal Perl HTML -> text conversion");
             require HTML::FormatText::WithLinks::AndTables;
             $formatter = \&_HTMLFormatText;
         } else {
@@ -1872,7 +1872,7 @@ sub _HTMLFormatter {
                 }
             }
 
-            RT->Logger->info("Using $prog for HTML -> text conversion");
+            RT->Logger->debug("Using $prog for HTML -> text conversion");
             $formatter = sub {
                 my $html = shift;
                 my $text = RT::Util::safe_run_child {
